README.md

```diff
@@ -40,29 +28,75 @@ In the source folder of your application, for Nano S:
 
 ```bash
 $ # docker can be replaced with podman or buildah without sudo
-$ sudo docker run --rm -ti -v "$(realpath .):/app" ledger-app-builder:latest
-root@656be163fe84:/app# make
+$ sudo docker run --rm -ti -v "$(realpath .):/app" --user $(id -u $USER):$(id -g $USER) ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
```

Comment:
No `--privileged -v "/dev/bus/usb:/dev/bus/usb"` ?
It's for loading apps I think

Reply:
Indeed, it is required for sideloading apps.

Reply:
Mentioned in next step "load apps". We should not add too many magic options rigth now.